js/notifications.js

```diff
@@ -33,13 +34,10 @@
             const numNotifications = this.length;
             console.log(
                 'Update notifications:',
-                'isFocused:', isFocused,
-                'isEnabled:', this.isEnabled,
-                'numNotifications:', numNotifications,
-                'shouldPlayNotificationSound:', shouldPlayNotificationSound
+                {isFocused, isEnabled, numNotifications, shouldPlayNotificationSound}
```

Comment:
I'm not convinced that this object will make it to the text logs as expected, though it might look alright in the browser.

Reply:
I double checked and it did properly get serialized as JSON which for simple structures is just as easy to read, and potentially even better as things are comma- instead of space-separated 😄